src/webkitgtk/mod.rs

```diff
@@ -242,6 +242,13 @@ impl InnerWebView {
       }
     }
 
+    // Extension loading
+    if let Some(extension_path) = pl_attrs.extension_path {
+      if let Some(extension_path) = extension_path.to_str() {
+        web_context.os.set_web_extensions_directory(extension_path);
+      }
+    }
```

Comment:
no need to convert to str here
```suggestion
    if let Some(extension_path) = &pl_attrs.extension_path {
      web_context.os.set_web_extensions_directory(extension_path);
    }
```